### ISSUE TITLE
Reordering is broken

### ISSUE THREAD

**Issue by jsilvestre** [jsilvestre/tasky#44]
An error pops in the console when a task is reordered.

**Comment by frankrousseau** [comment]
It's still doesn't work. Changes are done but not persisted.

**Comment by jsilvestre** [comment]
Indeed, thanks for the report (really need some tests to avoid that).
